databases/week3/meal_sharing.sql

```diff
@@ -0,0 +1,128 @@
+CREATE DATABASE hyf_lesson3
+    DEFAULT CHARACTER SET = 'utf8mb4';
+    use hyf_lesson3;
+    CREATE TABLE `meal` (
+  `id` int(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `title` VARCHAR(255) NOT NULL,
+  `description` TEXT NOT NULL,
+  `location` VARCHAR(255) NULL,
+  `when` DATETIME,
+  `max_reservations` INT(10) NOT NULL,
+  `price` DECIMAL NOT NULL,
```

Comment:
Remember to specify the number of digits and the number of decimal places that the decimal number can have.

Reply:
ok